src/components/step-nav.scss

```diff
@@ -0,0 +1,234 @@
+@import "../base/config.scss";
+
+//------------------------------------------------------------//
+
+// * StepNav Component
+// * 1. Settings
+// * 2. Define Default StepNav
+// * 3. Size Modifier Mixin
+// * 4. Small Modifier
+// * 5. Titled Modifier
+
+// TODO : document global dependencies
+// TODO : Add Titled Modifier
+
+//------------------------------------------------------------//
+
+// * 1. Settings
+
+$StepNav-size: 60px;
+$StepNav-border: 4px;
+
+$StepNav-size--small: 40px;
+$StepNav-border--small: 2px;
+
+// * 2. Define Default StepNav
+
+.StepNav-steps {
+  padding-left: ($StepNav-size * 1.25);
+  position: relative;
+  @include clear();
+  padding-bottom: $vertical-block-spacing--small;
+  display: flex;
+}
+
+.StepNav-step {
+  position: relative;
+  overflow: hidden;
+  list-style: none;
+  display: block;
+  padding: 0;
+  flex: auto;
+
+  &:before {
+    content: '';
+    width: 100%;
+    display: block;
+    position: absolute;
+    background: $ui-divider;
+    height: $StepNav-border;
+    top: ($StepNav-size / 2);
+    right: ($StepNav-size * 1.25);
+  }
+
+  &:first-child {
+    position: absolute;
+    left: 0;
+    top: 0;
+    .StepNav-stepLink,
+    .StepNav-stepItem {
```

Comment:
yeah, that could just be stepItem and the enabled state could handle the rest - good call